neqo-transport/src/send_stream.rs

```diff
@@ -575,13 +590,28 @@ impl SendStream {
             priority: TransmissionPriority::default(),
             retransmission_priority: RetransmissionPriority::default(),
             retransmission_offset: 0,
+	    sendorder: None,
```

Comment:
Doesn't that imply separate sendOrders for each of High/Normal/Low?   We only have one

Reply:
We effectively have that now, since we're handling this all inside of SendStreams - we get write_frames invoked for each TransmissionPriority, and for each TransmissionPriority we look at the streams according to SendOrder